pkg/controller/servicemesh/webhooks/hacks.go

```diff
@@ -185,6 +187,20 @@ func createWebhookResources(ctx context.Context, mgr manager.Manager, log logr.L
 		os.Exit(0)
 	}
 
+	log.Info("Creating Maistra Operator webhook PrometheusRule")
+	monclient, _ := clientmonitoring.NewForConfig(mgr.GetConfig())
+	if _, err := monclient.PrometheusRules(operatorNamespace).Create(context.TODO(),
+		newPrometheusRule(operatorNamespace,
+			"maistra-operator-prometheusrule",
+			"sum without (smcp_namespace) (servicemesh_members)",
+			"cluster:servicemesh_members:sum"), metav1.CreateOptions{}); err != nil {
+		if errors.IsAlreadyExists(err) {
+			log.Info("Maistra Operator webhook PrometheusRule already exists")
+		} else {
+			return pkgerrors.Wrap(err, "error creating Maistra Operator webhook PrometheusRule")
```

Comment:
What if the cluster doesn't support the `PrometheusRule` resource? In that case, we probably shouldn't stop the operator from starting.

Reply:
Should we only log a warning that this `PrometheusRule` is not created?

Reply:
Yes, I think so. Log a warning + emit a warning event.

Reply:
Done